packages/strict-ui/src/sanitize.css.ts

```diff
@@ -0,0 +1,211 @@
+import { injectGlobal } from 'emotion'
+
+/*!
+ * Sanitize 2.0.0 (http://git.io/sanitize)
+ * Licensed under the MIT License.
+ */
+// Copied from https://cdnjs.cloudflare.com/ajax/libs/sanitize.css/2.0.0/sanitize.min.css
+injectGlobal`
```

Comment:
Maybe there's a way to approach this reset more dynamically, rather than a blanket global style? This way you'll be able to use `strict-ui` components alongside others.

So for instance in my `Stack` and `Inline` component, when it's rendered as an  `ol` or `ul` I make use of a shared reset:
```tsx
export const Stack: React.FC<StackProps> = ({ gap = 3, ...props }) => (
  <div
    {...props}
    sx={{
      /* …stack styles */
      /* …reset returns the following: */
      /* Reset: ensure direct child list-items render without bullets */
      "& > li": {
        listStyleType: "none",
        "&:before": {
          position: "absolute",
          content:
            '"\\200B"' /* Add zero-width space to prevent VoiceOver disable */,
        },
      },
    }}
  />
);
```

Might be worth at taking a look at how @markdalgleish tackled this in Braid!
https://twitter.com/markdalgleish/status/1223161650246602752